sofl/onlinefix_game.py

```
@@ -77,258 +78,320 @@ def launch(self) -> None:
+            return
         dll_overrides = shared.schema.get_string("online-fix-dll-overrides")
-        
-        # Проверяем, запущен ли Steam (с учетом flatpak)
+
+        # Detect Flatpak environment and, if present, try to run host checks
+        in_flatpak = os.path.exists("/.flatpak-info")
+        host_home = os.path.expanduser("~")
+        if in_flatpak:
+            try:
+                host_home_proc = subprocess.run(
+                    ["flatpak-spawn", "--host", "bash", "-lc", "echo $HOME"],
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)